src/chart.js

```diff
@@ -106,24 +106,24 @@ var HPChart = CreateClass({
             }
         }
 
-        if (d == null) return; // ここでdがnullはあり得ないが保守的にreturns
-
-        var data = d[this.state.sortKey];
-        var csvData = [];
-        var header = [];
-        header.push(data[0][0]);
-        for (var i = 1; i < data[0].length; i++) {
-            // 不要な改行が見出しに含まれておりCSVが崩れるので消す
-            header.push(data[0][i].replace(/\r?\n/g, ""));
-        }
-        csvData.push(header);
-        // 2行目以降がデータ
-        for (var i = 1; i < data.length; i++) {
-            csvData.push(data[i]);
+        // where d can not be null but conservatively returns
+        if (d != null) {
+            var data = d[this.state.sortKey];
+            var csvData = [];
+            var header = [];
+            header.push(data[0][0]);
+            for (var i = 1; i < data[0].length; i++) {
+                // 不要な改行が見出しに含まれておりCSVが崩れるので消す
```

Comment:
// Remove unnecessary line feed letters which break CSV format.

Reply:
missed that one